paddle/majel/test/test_framework.cpp

```diff
@@ -0,0 +1,6 @@
+#include "gtest/gtest.h"
```

Comment:
Why we need this file? I think we can just link `-lgtest_main` to have the main function?